File: app/src/main/java/libertypassage/com/app/models/UserDetailsTrackUser.java

package libertypassage.com.app.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UserDetailsTrackUser {

    @SerializedName("usr_id")
    @Expose
    private Integer usrId;
    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("current_status")
    @Expose
    private Integer currentStatus;
    @SerializedName("risk_score")
    @Expose
    private Integer riskScore;
    @SerializedName("user_status")
    @Expose
    private UserStatus userStatus;

    public Integer getUsrId() {
        return usrId;
    }

    public void setUsrId(Integer usrId) {
        this.usrId = usrId;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public Integer getCurrentStatus() {
        return currentStatus;
    }

    public void setCurrentStatus(Integer currentStatus) {
        this.currentStatus = currentStatus;
    }

    public Integer getRiskScore() {
        return riskScore;
    }

    public void setRiskScore(Integer riskScore) {
        this.riskScore = riskScore;
    }

    public UserStatus getUserStatus() {
        return userStatus;
    }

    public void setUserStatus(UserStatus userStatus) {
        this.userStatus = userStatus;
    }

}

File: app/src/main/java/libertypassage/com/corporate/model/UserDetailsTrackUser.kt
package libertypassage.com.corporate.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UserDetailsTrackUser {
    @SerializedName("usr_id")
    @Expose
    var usrId: Int? = null

    @SerializedName("username")
    @Expose
    var username: String? = null

    @SerializedName("current_status")
    @Expose
    var currentStatus: Int? = null

    @SerializedName("risk_score")
    @Expose
    var riskScore: Int? = null

    @SerializedName("user_status")
    @Expose
    var userStatus: UserStatus? = null
}